examples/mobile/UIComponents/components/textfields/textfieldswithoutlabels.html

```diff
@@ -13,34 +13,35 @@
 	<div class="ui-page" id="textinput-demo">
 		<header>
 			<h1>
-				Text fields
+				Text fields without labels
 			</h1>
 		</header>
 		<!-- /header -->
 		<div class="ui-content">
 			<ul class="ui-listview ui-content-area">
-				<li class="ui-li-static">
-					<input placeholder="Hint text Inactive" type="text"/>
+				<li class="ui-li-text-input">
```

Comment:
It think "ui-li-has-text-input" is more proper to sync with other component such as 'ui-li-has-checkbox', or 'ui-li-has-icon'

Reply:
right, fixed